buildconfig/dev-packages/deb/mantid-developer/ns-control

```diff
@@ -61,6 +61,7 @@ Depends: git,
   python-h5py,
   python-yaml,
   python-mock,
+  python-psutil,
```

Comment:
And `python3-psutil`?

Reply:
Thanks. I'd forgotten we'd added the python3 dependencies now.